src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Collaborate with your teammates
+tags:
+  - New Relic 
+  - Use New Relic
+  - Get started
+metaDescription: "Our new collaboration features help you use New Relic to solve problems together"
+---
+
+import close_and_reopen_discussions from 'images/close-and-reopen-discussions.png'
+
+import collaboration_discussion_board_new from 'images/collaboration-discussion-board-new.png'
+
+import green_dot from 'images/green_dot.png'
+
+<Callout variant="important">
+The functionality described below is currently in preview. If you don't yet have access but would like to test the commenting and discussion functionality [please sign up here](https://one.newrelic.com/discussions).
+</Callout>
+
+We're making it easier to collaborate with your teammates by building discussion and commenting tools into New Relic. You can now add comments on any page. Our integrations with Slack, MS Teams (coming soon), and email let you share links and screenshots, and communicate with your teammates directly from the New Relic UI without switching applications. Our collaboration tools allow you to identify problems and find solutions in one place. Moving the discussion closer to your data streamlines your workflows, improves efficiency, and reduces MTTR.
+
+## How it works [#collaborate]
+
+<Video
+ type="wistia"
+ id="m4b6y854wi"
+/>
+
+We've integrated discussion tools directly into New Relic so you can: 
+
+* **Start discussions in context at the source.** When you comment on a dashboard or an error in New Relic, that comment stays with that dashboard or error.
+* **Sync discussions with Slack.**  With our Slack integration you can share links, screenshots, and comments to Slack in seconds. All teammates  can reply from Slack and discussions stay in sync, no matter where teammates contribute. [Learn more about the Slack integration](https://docs-preview.newrelic.com/docs/collaboration-slack-integration).
+* **Create shared knowledge and history.** Collaborating within the New Relic platform will allow your team to build a searchable body of shared knowledge.
+## Comment on any page [#comment-on-any-page]
+
+On the global navigation, there's a new “add comment” icon on most pages. A green dot on the icon means there is actually an open discussion on the page. A gray dot means there are closed discussions that can perhaps answer new questions or help resolve current issues. 
```

Comment:
I think the last two sentences should be consolidated and tweaked a bit, something like "A green dot on the icon means there's an open discussion on the page, while a gray dot indicates a closed discussion that may have information to help resolve a current issue." What do you think?